.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: cpp
+compiler: gcc
+sudo: require
+dist: trusty
+
+script:
+  - ./autogen.sh # Undocumented
+  - ./configure --prefix=/usr
+  - make -j$(nproc)
+  - make install DESTDIR=$(readlink -f appdir) ; find appdir/
+  - rm -rf appdir/usr/include/ appdir/usr/lib/pkgconfig/ appdir/usr/lib/*.{a,la} # Don't bundle developer stuff
+  - sudo mv appdir/usr/lib/* /usr/local/lib/ # Workaround for https://github.com/probonopd/linuxdeployqt/issues/160
+  - mkdir -p appdir/usr/share/icons/hicolor/128x128/apps ; cp dist/qpdf.png appdir/usr/share/icons/hicolor/128x128/apps/
+  - mkdir -p appdir/usr/share/applications ; cp dist/qpdf.desktop appdir/usr/share/applications/
+  - wget -c "https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage"
```

Comment:
What is the purpose of the qt stuff? Is it used elsewhere in AppImage, or are you just using it for functionality around bundling additional dependent libraries (jpeg and zlib)?

Reply:
`linuxdeployqt` is the tool that does all the work of bundling libraries and making the AppDir, then the AppImage. It uses Qt internally. Not Qt will end up in the AppImage.